### PR TITLE
[Feature 134] Profile Picture Display Implementation

### LINKED ISSUE #134: [Feature 134] shows profile photo on the view profile screen

### PR DESCRIPTION
**Pull request by candidcalcifer**
What: added functionality to display profile picture of each user, can be improved in the future by implementing a working library to convert the string variable to the correct drawable id, just not enough time for this deadline

Why: users want to see images of potential matches 

Implementation details: added png and jpg files into resources folder, implemented logic in view profiles fragments to check for and display the correct image

### CODE REVIEW

**Comment by jenniferjinyoungyang** [comment]
looks so good

File: app/src/main/java/com/courseproject/tindar/ui/home/SecondViewProfileFragment.java [hunk 1]
```diff
@@ -184,33 +189,26 @@ public void onDestroyView() {
...
-            profilePicture.setImageBitmap(result);
-        }
-    }
 
     private void setsProfile() {
         ViewProfileResponseModel profile =
                 viewProfilesController.getProfile(otherUserId);
 
+        profileLink = profile.getProfilePictureLink();
+        if (profileLink.equals("spongebob1.png")){
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ui/home/SecondViewProfileFragment.java:198]: can you comment on it right above this line 
// This is only a temporary implementation for demo purposes. The plan is to connect to show the picture from the external URL.

File: app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java [hunk 2]
```diff
@@ -182,36 +183,25 @@ public void onDestroyView() {
...
-            profilePicture.setImageBitmap(result);
-        }
-    }
-
     private void setsProfile() {
         ViewProfileResponseModel profile =
                 viewProfilesController.getProfile(otherUserId);
 
+        profileLink = profile.getProfilePictureLink();
+        if (profileLink.equals("spongebob1.png")){
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java:191]: can you comment on it right above this line something like
// This is only a temporary implementation for demo purposes. The plan is to connect to show the picture from the external URL.

**Review by jenniferjinyoungyang — DISMISSED** [review]

File: app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java (hunk 2 above)
Comment (yiwxng) [reply app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java:191]: I wonder if a Switch statement would be good for this case

**Review by yiwxng — DISMISSED** [review]

**Comment by jenniferjinyoungyang** [comment]
I thinks one possible future plan can be enable user to upload the profile pic (and we can send it to some cloud and store it) and we can download back from that cloud to show it.

File: app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java (hunk 2 above)
Comment (candidcalcifer) [reply app/src/main/java/com/courseproject/tindar/ui/home/ViewProfileFragment.java:193]: i tried it but it wasn't loading the images sometimes so i switched it back to the if else

**Review by jenniferjinyoungyang — APPROVED** [review]

**Review by yiwxng — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Pull request merged**